components/molecules/client/user/component.tsx

```diff
@@ -0,0 +1,53 @@
+import FollowBtn from '@/components/molecules/client/follow-btn';
+import Image from 'next/image';
+
+interface UserProps {
+  userId: string;
+  postId: string;
+  initialFollowed: boolean;
+  name: string;
+  followersCount: number;
+  bio: string;
+  avatarUrl: string;
+}
+
+const UserComponent: React.FC<UserProps> = ({
+  userId,
+  name,
+  followersCount,
+  bio,
+  avatarUrl,
+}) => {
+  return (
+    <div className="flex items-center justify-between w-[460px] h-[89px] gap-md p-4 border border-light-neutral-gray rounded-md">
+      <div className="flex items-center gap-md">
+        {/* Image Box with Fixed Size and Ellipse Shape */}
+        <div className="w-[50px] h-[50px] rounded-full overflow-hidden flex-shrink-0">
+          <Image
+            src={avatarUrl}
+            alt={name}
+            width={50}
+            height={50}
+            className="w-full h-full object-cover"
+          />
+        </div>
+        <div className="flex flex-col justify-center gap-sm w-[329px]">
```

Comment:
Remove "gap-sm" class

Reply:
also replace "w-[329px]" with "flex-1". try not to hardcode spacing values. use -xs -sm -md -lg or -xl